diff --git a/node_modules/gatsby-plugin-netlify/constants.js b/node_modules/gatsby-plugin-netlify/constants.js
index 73b5a18..f0df677 100644
--- a/node_modules/gatsby-plugin-netlify/constants.js
+++ b/node_modules/gatsby-plugin-netlify/constants.js
@@ -32,8 +32,10 @@ const SECURITY_HEADERS = {
 exports.SECURITY_HEADERS = SECURITY_HEADERS;
 const IMMUTABLE_CACHING_HEADER = `Cache-Control: public, max-age=31536000, immutable`;
 exports.IMMUTABLE_CACHING_HEADER = IMMUTABLE_CACHING_HEADER;
+const NO_CACHING_HEADER = `Cache-Control: public, max-age=0, must-revalidate`;
 const CACHING_HEADERS = {
   "/static/*": [IMMUTABLE_CACHING_HEADER],
+  "/static/d/*": [NO_CACHING_HEADER],
   "/sw.js": [`Cache-Control: no-cache`]
 };
 exports.CACHING_HEADERS = CACHING_HEADERS;
